hivemind/moe/server/runtime.py

```diff
@@ -108,6 +104,11 @@ def run(self):
                 if not self.shutdown_trigger.is_set():
                     self.shutdown()
 
+    def process_batch(self, pool: TaskPoolBase, batch_index: int, *batch: torch.Tensor) -> Tuple[Any, int]:
```

Comment:
I think `outputs` is `List[torch.Tensor]` due to https://github.com/learning-at-home/hivemind/blob/master/hivemind/moe/server/task_pool.py#L241